.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+    env: {
+        browser: true,
+        es6: true,
+        node: true
+    },
+    extends: "eslint:recommended",
+    parserOptions: {
+        ecmaVersion: 2015,
+        sourceType: "module"
+    },
+    rules: {
+        "no-console": "off"
+    }
+};
```

Comment:
Think this is the only file with 4 space indentation, could it be switch to 2 spaces for consistency :)